### DISCUSSION TITLE
Clicking on a publication or portfolio link doesn't generate any content

### DISCUSSION THREAD

**Discussion by johnli25** [academicpages/academicpages.github.io#2369] [Q&A]
The original GitHub academic page portfolio home page: https://academicpages.github.io/portfolio/. When clicking on a publication or portfolio link (which redirects to https://academicpages.github.io/portfolio/portfolio-1/) has a nicely formatted Markdown page of content:
<img width="1298" alt="image" src="https://github.com/user-attachments/assets/935bab04-d3e2-4183-b440-93fd3107c888">

However, for my github.io repo portfolio home page (https://johnli25.github.io/portfolio/), when I click on a publication or portfolio link (https://johnli25.github.io/portfolio/portfolio-1/), all the nice formatting goes away-the sidebar content disappears, no "titles", no previous/next button options, and no options to share on social media. 
<img width="1367" alt="image" src="https://github.com/user-attachments/assets/9a9469ea-c1a0-4cd9-a8c6-ba272f2e4580">

Any idea on what might be going on? Is this a bug?

**Comment by rjzupkoii** [comment]
That doesn't look like an issue with the template so much as a configuration issue. I took a quick look at your repository but since there's been a lot of changes in the past couple day's it's hard to quickly tell what the root cause might be.

**Reply by johnli25** [reply]
Thank you @rjzupkoii! Can you elaborate what area this configuration issue might be in?

**Reply by rjzupkoii** [reply]
Not sure to be honest, in cases this like I usually start by going through the recent commits to see what has changed.
